src/cli/templates/init/botfile.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
    */
   formsDir: './forms',
 
+  /*
+    Path to Content Forms Data
+   */
+  formsDataDir: './forms_data',
```

Comment:
Could we use the `dataDir` (+ `./forms`) for it? I'd like to make it easy for people to swap all data location in a breeze

Reply:
My idea was to have parallel structures in `forms` and `forms_data` to simplify editing and reduce mental overhead. Can change of course

Reply:
Ah, data is not the best place for it I'm afraid. Data is gitignored while these serialized forms should be part f the version-controlled code, that's the whole idea of ghost content in the end, right?

Reply:
oh yeah, you're right `data` is git ignored – haven't thought about that. Let's keep `forms_data` as I can't think of a better place right now